sorc/ncep_post.fd/CALCHIPSI.f

```diff
@@ -0,0 +1,176 @@
+!> @file
```

Comment:
@KarinaAsmar-NOAA @JesseMeng-NOAA Is this new algorithms from George  Vandenberghe?

Reply:
This infrastructure of this subroutine does not match that of UPP and UFS MPI code.